students/kulon_aleksander/lesson_05/greater_than_neighbours.py

```diff
@@ -0,0 +1,15 @@
+def func():
```

Comment:
Have a look at the code below. We should omit first and last element. Next step is to check if neighbours from both sides are smaller, if yes increase quantity.

```
def func():
    quantity = 0
    a = input().split()
    for i in range(1, len(a) -1):
        if int(a[i-1]) < int(a[i]) > int(a[i+1]):
            quantity += 1
    print(quantity)


def main():
    func()


if __name__ == '__main__':
    main()
```

Reply:
mi sie pomylilo i zle zadanie ze sneakify wklejalem, wklejalem "greater than previous" zamiast "neightbours" :D